### PR TITLE
chore(nifi): add brokerCACert param to subscription params

### PR DESCRIPTION
**Pull request by subirjolly**
Related Issue: https://github.com/influxdata/data-acquisition/issues/441

add `brokerCACert` and `authType` params as we need 4 params
1. `brokerCACert`
2. `brokerClientCert`
3. `brokerClientKey`

### CODE REVIEW

**Review by helenosheaa — APPROVED** [review]

**Comment by helenosheaa** [comment]
we should add/change these with this PR:

- `brokerClientCert`
- `brokerClientKey`
- `brokerCACert`
- `brokerKeypairVaultPath`
- `authType`

**Comment by genehynson** [comment]
> * brokerKeypairVaultPath

This wouldn't be included in the API contract, right? The other ones make sense to me.

**Comment by helenosheaa** [comment]
> This wouldn't be included in the API contract, right? The other ones make sense to me.

correct good catch, i'm copy pasting the whole list, edited :)

**Review by helenosheaa — APPROVED** [review]
lgtm

**Review by genehynson — APPROVED** [review]

File: src/svc/nifid/schemas/Subscription.yml [hunk 1]
```diff
@@ -19,6 +19,15 @@ properties:
     type: string
   topic:
     type: string
+  certCreatedAt:
```
Comment (genehynson) [inline src/svc/nifid/schemas/Subscription.yml:22]: nit: `certProvidedAt` since we're not creating the certificate

**Review by genehynson — APPROVED** [review]

**Pull request merged**